deny.toml

```diff
@@ -59,6 +59,19 @@ ignore = [
     "RUSTSEC-2022-0055",
     "RUSTSEC-2023-0001",
     "RUSTSEC-2022-0084",
+    "RUSTSEC-2023-0005",
+    "RUSTSEC-2022-0070",
+    "RUSTSEC-2019-0036",
+    "RUSTSEC-2022-0041",
+    "RUSTSEC-2022-0078",
+    "RUSTSEC-2021-0145",
+    "RUSTSEC-2020-0056",
+    "RUSTSEC-2022-0080",
+    "RUSTSEC-2020-0036",
+    "RUSTSEC-2021-0139",
+    "RUSTSEC-2021-0059",
+    "RUSTSEC-2021-0060",
+    "RUSTSEC-2022-0090",
```

Comment:
We usually add comments about each advisory, but since this is a long list you can add a todo instead.

Reply:
I saw we don't add the comments lately, so I directly add those without comments.

Further information can be found in advisory db anyway. Like https://rustsec.org/advisories/RUSTSEC-2022-0090.html